spec/integration_spec.rb

```diff
@@ -88,7 +88,7 @@
       behaves_like cli_spec 'document_alamofire',
                             '-m Alamofire -a Alamofire ' \
                             '-u https://nshipster.com/alamofire ' \
-                            '-x -project,Alamofire.xcodeproj ' \
+                            '-x -project,Alamofire.xcodeproj,-dry-run ' \
```

Comment:
using `-dry-run` by default won't work for cases where there are non-apple dependencies, but that's not the case for either Alamofire or MiscJazzyFeatures.